### PR TITLE
Update github action names

### PR DESCRIPTION
**Pull request by otherview**
### Why is this change needed?

- Updating actions names

### CODE REVIEW

File: .github/workflows/manual-deploy-testnet.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # WebSocket:  9000, 9001, 9002
 #
 
-name: Manual Deploy Obscuro network
+name: '[M] Deploy Testnet'
```
Comment (tudor-malene) [inline .github/workflows/manual-deploy-testnet.yml:10]: maybe add L2 - for consistency

**Review by tudor-malene — APPROVED** [review]
lgtm ( added a suggestion)

**Review by BedrockSquirrel — APPROVED** [review]
LGTM, agree with Tudor's comment

File: .github/workflows/manual-deploy-testnet.yml (hunk 1 above)
Comment (otherview) [reply .github/workflows/manual-deploy-testnet.yml:10]: Sure 👍

**Pull request merged**